tlatools/org.lamport.tlatools/src/tla2sany/explorer/Explorer.java

```diff
@@ -59,11 +58,13 @@ public class Explorer {
 	private ExploreNode obj;
 
 	private ExternalModuleTable mt;
+	private Errors errors;
```

Comment:
Nit: this field can be `final`.

(So can many other fields in this class, but let's keep things focused on `Errors`.)